source/_docs/bots-and-indexing.md

```diff
@@ -50,13 +50,13 @@ You can index your site under your production domain once it's added to the Live
 
 ## Troubleshooting
 
-### Google Links to My Site over HTTPS with Pantheon's Certificate
+### Google Results for My Site Link to Pantheon's Platform Domain Using HTTPS

```

Comment:
They link to the customer's Domain, but using the `*.pantheonsite.io` certificate.